### DISCUSSION TITLE
Easy way to install latest version of Spine Toolbox

### DISCUSSION THREAD

**Discussion by ghost** [Spine-project/Spine-Toolbox#1575] [Show and tell]
Check out https://github.com/Spine-project/spinetoolbox-dev

**Comment by ghost** [comment]
if no comments or objections, I’ll add this to Toolbox’s README as well.

**Reply by ptsavol** [reply]
So, these new instuctions would be added as the third installation option (along with venv and conda) into chapter https://github.com/Spine-project/Spine-Toolbox#installation-from-sources-using-git, right?

**Reply by ghost** [reply]
I’m thinking something like this under **Installation with Python and pipx**:
```diff
5. Install the latest Spine Toolbox release by running

        python -m pipx install spinetoolbox

-That’s it!
-To launch Spine Toolbox, open a terminal and run
+   or get the latest development version using
+
+        python -m pipx install git+https://github.com/Spine-project/spinetoolbox-dev
+
+That’s it! To launch Spine Toolbox, open a terminal and run

    spinetoolbox

**Reply by ptsavol** [reply]
Ok sounds good, but don't forget to add the `spinetoolbox-dev` upgrade instructions as well.

**Reply by ghost** [reply]
Yep, I have 

```
To update Spine Toolbox to the latest available release, open a terminal and run

    python -m pipx upgrade spinetoolbox

Here, replace `spinetoolbox` with `spinetoolbox-dev` if you installed the latest
development version.
```
there.
